### PR TITLE
[DTL] Update DTL (Diff Template Library) to v1.20

### PR DESCRIPTION
**Pull request by bobk2824**
Version 1.20 of the library is available. I'm updating vcpkg to pull the latest version.

- #### What does your PR fix?
no fixes

- #### Which triplets are supported/not supported? Have you updated the [CI baseline]
no change - this is a header-only template library.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
- Yes

### CODE REVIEW

File: ports/dtl/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "name": "dtl",
-  "version-string": "1.19",
-  "port-version": 1,
+  "version": "1.20",
   "description": "Diff template library"
```
Comment (Adela0814) [inline ports/dtl/vcpkg.json:4]: Please add license for `dtl`, thanks.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

File: ports/dtl/vcpkg.json (hunk 1 above)
Comment (bobk2824) [reply ports/dtl/vcpkg.json]: Added license. Thanks

File: ports/dtl/portfile.cmake [hunk 2]
```diff
@@ -4,8 +4,8 @@ include(CMakePackageConfigHelpers)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO cubicdaiya/dtl
-    REF v1.19
-    SHA512 77c767451b1b78ce49085da6ff5bb8a23c96dec56a37d96ef357a6b69a1b2cd45e2c6c4e8f91ee34ca080ce03a26518c478ff207309326a4bc7e729eaa2824b2
+    REF v1.20
+    SHA512 44cdaf190d8a103effbca8df244c652b642590795f7307f5f7fdf64fc34bdbe2fa5ab2e1a08185abf099e35b0d9158306a80a8dc24bba9eccab4c77c7b1eed5e
     HEAD_REF master
 )
```
Comment (Adela0814) [inline ports/dtl/portfile.cmake:11]: Please quote the absolute path below.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

File: ports/dtl/portfile.cmake (hunk 2 above)
Comment (Adela0814) [reply ports/dtl/portfile.cmake:11]:
```
file(INSTALL "${SOURCE_PATH}/${PORT}"
  DESTINATION "${CURRENT_PACKAGES_DIR}/include")
file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION
  "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```

**Review by Adela0814 — CHANGES_REQUESTED** [review]

**Review by Adela0814 — APPROVED** [review]

**Pull request merged**